src/main/java/step2/Operator.java

```diff
@@ -0,0 +1,40 @@
+package step2;
+
+public enum Operator {
+    PLUS("+") {
+        @Override
+        int calculate(Number number1, Number number2) {
+            return number1.getValue() + number2.getValue();
+        }
+    }, MINUS("-"){
+        @Override
+        int calculate(Number number1, Number number2) {
+            return number1.getValue() - number2.getValue();
+        }
+    }, MULTIPY("*") {
+        @Override
+        int calculate(Number number1, Number number2) {
+            return number1.getValue() * number2.getValue();
+        }
+    }, DEVIDE("/") {
+        @Override
+        int calculate(Number number1, Number number2) {
+            return number1.getValue() / number2.getValue();
+        }
+    };
+
+    private String value;
+
+    Operator(String value) {
+        this.value = value;
+    }
+
+    public boolean isEqualTo(String value) {
+        if(this.value.equals(value)) {
+            return true;
+        }
+        return false;
```

Comment:
```suggestion
        return this.value.equals(value);
```
한줄로 이렇게 쓰는게 더 좋을 것 같아요 :)